### PR TITLE
Added section images

### PR DESCRIPTION
**Pull request by jspaaks**
To separate the content a bit better, I added section images. Hope you like it!

PS
When making these changes, I noticed that perhaps it's good to

1. put the Swagger web service before the Flask app 
1. make the first part of the webassembly section result in a service instead of a HTML page

### CODE REVIEW

File: images/flask.svg [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+           style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:40px;line-height:125%;font-family:Akkurat-Bold;-inkscape-font-specification:'Akkurat-Bold Bold';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+           transform="matrix(0.26458333,0,0,0.26458333,34.146994,176.78145)"><flowRegion
+             id="flowRegion1002-3"><rect
+               id="rect1004-73"
+               width="932.27899"
+               height="415.6915"
+               x="1267.2537"
+               y="102.80756" /></flowRegion><flowPara
+             id="flowPara1006-2"
+             style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:53.33333588px;font-family:'Roboto Condensed';-inkscape-font-specification:'Roboto Condensed, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;fill:#333333">two-way C++ to Python bindings</flowPara></flowRoot>        <flowRoot
```
Comment (sverhoeven) [inline images/flask.svg:136]: Could you remove `two way` we are just calling C++ from Python and we are not calling Python from C++.

```suggestion
             style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:53.33333588px;font-family:'Roboto Condensed';-inkscape-font-specification:'Roboto Condensed, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;fill:#333333">C++ to Python bindings</flowPara></flowRoot>        <flowRoot
```

Same goes for other occurences

File: images/react.svg [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+         id="flowRoot1000-3"
+         xml:space="preserve"><flowRegion
+           id="flowRegion1002-6"><rect
+             y="102.80756"
+             x="1267.2537"
+             height="415.6915"
+             width="932.27899"
+             id="rect1004-7" /></flowRegion><flowPara
+           style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:53.33333588px;font-family:'Roboto Condensed';-inkscape-font-specification:'Roboto Condensed, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;fill:#333333"
+           id="flowPara1006-5">Emscriptem</flowPara></flowRoot>      <flowRoot
```
Comment (sverhoeven) [inline images/react.svg:141]: Typo
```suggestion
           id="flowPara1006-5">Emscripten</flowPara></flowRoot>      <flowRoot
```

File: images/wasm.svg [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+         id="flowRoot1000-3"
+         xml:space="preserve"><flowRegion
+           id="flowRegion1002-6"><rect
+             y="102.80756"
+             x="1267.2537"
+             height="415.6915"
+             width="932.27899"
+             id="rect1004-7" /></flowRegion><flowPara
+           style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:53.33333588px;font-family:'Roboto Condensed';-inkscape-font-specification:'Roboto Condensed, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;fill:#333333"
+           id="flowPara1006-5">Emscriptem</flowPara></flowRoot>    </g>
```
Comment (sverhoeven) [inline images/wasm.svg:141]: Typo
```suggestion
           id="flowPara1006-5">Emscripten</flowPara></flowRoot>    </g>
```

File: images/flask.svg (hunk 1 above)
Comment (jspaaks) [reply images/flask.svg:136]: 'two ways' removed in next commit

File: images/react.svg (hunk 2 above)
Comment (jspaaks) [reply images/react.svg:141]: typo corrected in next commit

File: images/wasm.svg (hunk 3 above)
Comment (jspaaks) [reply images/wasm.svg:141]: typo corrected in next commit

**Comment by jspaaks** [comment]
> for Python we should add pybind11 in image.

I thought Python binding was a bit more generic, therefore I left it like this. Let me know if you'd still like to see it changed.

**Comment by sverhoeven** [comment]
> > for Python we should add pybind11 in image.
> 
> I thought "Python bindings" was a bit more generic, therefore I left it like this. Let me know if you'd still like to see it changed.

For consistencies sake we should have the flavor of bindings and the bindings library in both JS and Python diagrams. Or just have the flavor of bindings.

**Comment by jspaaks** [comment]
I think it's OK to sacrifice some consistency across the document if that makes an individual section image easier to understand. The purpose of the images is not to accurately capture all the details that are discussed in the corresponding text; rather, they provide an anchor for readers to recognize. In that role, I thought that "C++ to Python bindings" will be recognized by more readers than "pybind11".

**Comment by jspaaks** [comment]
On svg v png: On GitHub everything works ok, but at least my editor (Atom) struggles with rendering order of layers, e.g. text on top of areas. png is a bit more rebust in that sense.

**Comment by jspaaks** [comment]
> dark theme in VS code

Same in atom. I'll add a white bg in the next round of edits after #66

**Pull request merged**